docs/extras/game_modding.md

```diff
@@ -7,9 +7,9 @@ LayeredFS, a tool built in to Atmosphere, allows you to replace game files with
 ### Instructions
 
 !!! tip ""
-    Make a folder on your SD card inside the `atmosphere/titles/` folder, with the name being the title ID of the game you would like to replace the files of. 
+    Make a folder on your SD card inside the `atmosphere/contents/` folder, with the name being the title ID of the game you would like to replace the files of. **Note: On Atmosphere 0.9.4 and below 'contents' is called 'titles`**
```

Comment:
Shouldn't it be 'titles' instead of 'titles`?

Reply:
whoops, you're right, sorry!